--- lua-5.1.5/src/lauxlib.c
+++ lua-5.1.5-patch/src/lauxlib.c
@@ -573,7 +573,3 @@
   if (c == LUA_SIGNATURE[0] && filename) {  /* binary file? */
-    lf.f = freopen(filename, "rb", lf.f);  /* reopen in binary mode */
-    if (lf.f == NULL) return errfile(L, "reopen", fnameindex);
-    /* skip eventual `#!...' */
-   while ((c = getc(lf.f)) != EOF && c != LUA_SIGNATURE[0]) ;
-    lf.extraline = 0;
+    return errfile(L, "sandbox", fnameindex);
   }